pkg/apis/monitoring/v1/thanos_types.go

```diff
@@ -250,6 +250,18 @@ type ThanosRulerSpec struct {
 	// operator itself) or when providing an invalid argument the reconciliation will
 	// fail and an error will be logged.
 	AdditionalArgs []Argument `json:"additionalArgs,omitempty"`
+	// Defines the configuration of the ThanosRuler web server.
+	Web *ThanosRulerWebSpec `json:"web,omitempty"`
+}
+
+// ThanosRulerWebSpec defines the configuration of the ThanosRuler web server.
+// +k8s:openapi-gen=true
+type ThanosRulerWebSpec struct {
+	WebConfigFileFields `json:",inline"`
+
+	// The Thanos Ruler web page title.
+	// +optional
+	PageTitle *string `json:"pageTitle,omitempty"`
```

Comment:
This isn't used anywhere since not supported by Thanos?
```suggestion
```

Reply:
I didn't find `pageTitle` in Thanos Ruler flags so I'll remove it.